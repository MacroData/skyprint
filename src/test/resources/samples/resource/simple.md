## My Message [/message]

### Retrieve a Message [GET]
This action has **two** responses defined

### Update a Message [PUT]


