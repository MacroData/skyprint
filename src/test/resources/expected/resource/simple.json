{
  "actions": [
    {
      "identifier": "Retrieve a Message",
      "template": null,
      "method": "GET",
      "description": "This action has **two** responses defined"
    },
    {
      "identifier": "Update a Message",
      "template": null,
      "method": "PUT",
      "description": ""
    }
  ],
  "identifier": "My Message",
  "template": "/message",
  "method": null,
  "description": ""
}
